public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -160,6 +206,98 @@ function Keyboard(menuUI) {
+                        $('#severity-button-1').click();
+                        svv.tracker.push("KeyboardShortcut_Severity1", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.noButton.hasClass('chosen')) {
+                        $('#no-button-1').click();
+                        svv.tracker.push("KeyboardShortcut_DisagreeReason1", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.unsureButton.hasClass('chosen')) {
+                        $('#unsure-button-1').click();
+                        svv.tracker.push("KeyboardShortcut_UnsureReason1", {
+                            keyCode: e.keyCode
+                        });
+                    }
+                    break;
+
+
+                case 50: // "2" key
+                case 98: // Numpad "2" key
+                    if (menuUI.yesButton.hasClass('chosen')) {
+                        $('#severity-button-2').click();
+                        svv.tracker.push("KeyboardShortcut_Severity2", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.noButton.hasClass('chosen')) {
+                        $('#no-button-2').click();
+                        svv.tracker.push("KeyboardShortcut_DisagreeReason2", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.unsureButton.hasClass('chosen')) {
+                        $('#unsure-button-2').click();
+                        svv.tracker.push("KeyboardShortcut_UnsureReason2", {
+                            keyCode: e.keyCode
+                        });
+                    }
+                    break;
+
+                case 51: // "3" key
+                case 99: // Numpad "3" key
+                    if (menuUI.yesButton.hasClass('chosen')) {
+                        $('#severity-button-3').click();
+                        svv.tracker.push("KeyboardShortcut_Severity3", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.noButton.hasClass('chosen')) {
+                        $('#no-button-3').click();
+                        svv.tracker.push("KeyboardShortcut_DisagreeReason3", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.unsureButton.hasClass('chosen')) {
+                        $('#unsure-button-3').click();
+                        svv.tracker.push("KeyboardShortcut_UnsureReason3", {
+                            keyCode: e.keyCode
+                        });
+                    }
+                    break;
+
+                // Shortcut to focus the comment text box.
+                case 52: // "4" key
+                case 100: // Numpad "4" key
+                    if (menuUI.yesButton.hasClass('chosen')) {
```

Comment:
I think that I'm happy with the number 4 focusing on the agree comment box. Can we also do that with the letter 'C' (for 'comment')?